### PR TITLE
About pages (mobile)

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the `/about` pages for mobile

## Description

- adds `/about` pages
- adds `AboutDescription`, `LatestGranteesList`, `AboutLayout` components
- updates `constants`, `theme` and `types`
- fetches the grants list data and displays it on `/about/who-we-support`
- moves `PROJECT_GRANTS_DOWNLOAD_FILE_URL` to env
- updates `package.json` w/required dependencies (to fetch, parse data and render markdown)

### CODE REVIEW

File: src/components/layout/AboutLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              </Tab>
+            ))}
+          </TabList>
+        </Tabs>
+      </Flex>
+
+      <Stack px={5}>
+        <Stack mb={8}>{children}</Stack>
+
+        {APPLICANTS_PAGES.some(path => router.pathname === path) && (
```
Comment (pettinarip) [inline src/components/layout/AboutLayout.tsx:107]: 🤔 is this necessary? is there a case were we are in the /about page and we display the applicants content?

File: src/pages/about/who-we-support.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import { LatestGranteesList } from '../../components/forms';
+import { PageSection, PageText } from '../../components/UI';
+
+import { CURRENT_GRANTS_QUARTERS, ESP_BLOG_URL } from '../../constants';
+
+import { Grant } from '../../types';
+
+// getStaticProps runs server-side only (on build-time)
+// https://nextjs.org/docs/basic-features/data-fetching/get-static-props#write-server-side-code-directly
+export const getStaticProps: GetStaticProps = async context => {
```
Comment (pettinarip) [inline src/pages/about/who-we-support.tsx:16]: 👍🏼

**Review by pettinarip — APPROVED** [review]
So fresh so clean. Leave a comment there but overall looks great.

File: src/components/forms/constants.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { list } from '@chakra-ui/react';
+import { AboutDescription } from './../UI/common/AboutDescription';
```
[the remark this answers is not in the document]
Comment (nhsz) [reply src/components/forms/constants.ts:2]: not used, removed!

File: src/components/layout/AboutLayout.tsx (hunk 1 above)
Comment (nhsz) [reply src/components/layout/AboutLayout.tsx:107]: Removed! I forgot to delete that. I moved the `ReadyToApply` component where t's necessary to handle better the desktip layout.

**Pull request merged**